src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -352,7 +352,7 @@ configurationRegistry.registerConfiguration({
 		},
 		'search.experimental.notebookSearch': {
 			type: 'boolean',
-			description: nls.localize('search.experimental.notebookSearch', "Controls whether to use the experimental notebook search in the global search. Please refresh your search for changes to this setting to take effect."),
+			description: nls.localize('search.experimental.notebookSearch', "Controls whether to use the experimental notebook search in the global search. Please refresh VS Code instance for changes to this setting to take effect."),
```

Comment:
it now uses a different search input if you use the experiment, so you need to reload the window

Reply:
You should say "reload the window" here's, it's not clear what refreshing means.